docs/Migrating-v4.md

```diff
@@ -365,24 +361,31 @@ Variables provided:
 - typgraphy
 - shadow
 
-By default, these variables take values from the SASS variables. You can freely override them in a custom CSS file loaded after ECL one.
-
-Note: we are using shorter aliases internally to limit the size of the compiled css file, but it is recommended to use the full variable name if you want to override them
-
-### How to customize component display
+By default, these variables take values from the SASS variables. You can freely override any variable you want in a custom CSS file loaded after ECL one.
 
-[WIP]
+Example:
 
-Every component is now exposing a set of variables, managing different aspect of their display. These variables can be overriden to alter the look and feel of the component, by picking values defined in the styles.
+```
+your_custom_css_file.css
 
-There are two ways to customize values related to the components:
+:root {
+  --ecl-color-primary: #00f;
+  --ecl-color-primary-120: #00d;
+  ...
+  --ecl-font-m: normal normal 400 1rem/1.75rem "Comic Sans ms",arial,sans-serif;
+  ...
+  --ecl-shadow-1: 1px 1px 2px rgba(0, 0, 0, 0.5);
+  ...
+}
+```
 
-#### Using SASS variables
+Note: we are using shorter aliases internally to limit the size of the compiled css file, but it is recommended to use the full variable name if you want to override them
 
-SASS variables are defined in the file `src/themes/[your_theme]/_variables.scss`. There are two theme available by default (EC and EU), which could act as example.
+### Using SASS variables
 
-You can edit them in your custom theme, and recompile ECL css by running the command `yarn dist` at the root of the project. The compiled files are available in `/dist/packages/[your_theme]`
+If you are building ECL directly on your side, you can change the values of the SASS variables before compilation.
+This is not the recommended way to customize ECL, as it may conflict with future updates (CSS variables are prefered), but this is an option if you have full control over the ECL implementation.
 
-#### Using CSS variables
+SASS variables are defined in the files `src/themes/[your_theme]/variables/*.scss`. There are two theme available by default (EC and EU).
 
-[To be done]
+You can edit the values, and recompile ECL css by running the command `yarn dist` at the root of the project. The compiled files are available in `/dist/packages/[your_theme]`
```

Comment:
i
```suggestion
You can edit the values, and recompile ECL css by running the command `yarn` at the root of the project. The compiled files are available in `/src/presets/[ec/eu]/dist`.
```
I think that suggesting to run yarn dist only to get the resources compiled is an overkill, running yarn is enough to get them compiled and they can be retrieved from the presets folders.